core/src/main/java/io/aiven/klaw/helpers/db/rdbms/InsertDataJdbc.java

```diff
@@ -626,4 +562,25 @@ public String insertProductDetails(ProductDetails productDetails) {
     productDetailsRepo.save(productDetails);
     return ApiResultStatus.SUCCESS.value;
   }
+
+  public void insertIntoActivityLog(
+      String requestType,
+      int tenantId,
+      String operationType,
+      int teamId,
+      String details,
+      String envId,
+      String requestor) {
+    ActivityLog activityLog = new ActivityLog();
+    activityLog.setReq_no(getNextActivityLogRequestId(tenantId));
+    activityLog.setActivityName(requestType + "Request");
```

Comment:
Does this "Request" need a space to separate the words e.g. " Request"

Reply:
Good one. Will do.